client/src/Movies/MovieList.js

```diff
@@ -32,23 +33,25 @@ export default class MovieList extends Component {
 }
 
 function MovieDetails({ movie }) {
-  const { title, director, metascore, stars } = movie;
+  const { title, director, metascore, stars, id } = movie;
```

Comment:
You just need to move this function into `MovieCard.js`, pretty straight forward stretch requirement